_layouts/landing.html

```diff
@@ -119,24 +119,41 @@ <h5 class="title">Reference</h5>
-                deploy your applications using Docker.
-              </p>
             </div>
           </div>
           <div class="row">
-            <div class="col-xs-12 col-md-6"><a href="/language/">Containerize language-specific apps using Docker</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/engine/reference/builder/">Write a Dockerfile</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/network/">Manage container networking</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/compose/compose-file/">Write a Docker Compose file</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/storage/">Work with volumes and bind mounts</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/get-started/04_sharing_app/">Share my image on Docker Hub</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/config/daemon/">Configure the Docker daemon</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/get-started/02_our_app/">Build and run an image</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/config/labels-custom-metadata/">Manage Docker objects</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/build/building/multi-stage/">Multi-stage builds</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/get-started/swarm-deploy/">Scale apps using Swarm</a></div>
+            <ul class="nav nav-tabs">
+              <li class="active"><a data-toggle="tab" data-target="#includes">Explore Docker</a></li>
+              <li><a data-toggle="tab" data-target="#features">Get Support</a></li>
+              <li><a data-toggle="tab" data-target="#faqs">Browse FAQs</a></li>
+              </ul>
+              <div class="tab-content">
+              <div id="includes" class="tab-pane fade in active" markdown="1">
+               <div class="col-xs-12 col-md-6"><a href="/engine/reference/builder/">Write a Dockerfile</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/network/">Manage container networking</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/compose/compose-file/">Write a Docker Compose file</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/storage/">Work with volumes and bind mounts</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/get-started/04_sharing_app/">Share my image on Docker Hub</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/config/daemon/">Configure the Docker daemon</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/get-started/02_our_app/">Build and run an image</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/config/labels-custom-metadata/">Manage Docker objects</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/build/building/multi-stage/">Multi-stage builds</a></div>
+              </div>
+              <div id="features" class="tab-pane fade" markdown="1">
```

Comment:
The links point directly to Docker Desktop GH repositories. Suggest we point users to https://docs.docker.com/support/ , Community forums, Community slack

Reply:
Updated